src/System.Private.CoreLib/src/Microsoft/Win32/Win32Native.cs

```diff
@@ -165,7 +165,6 @@ internal static class Win32Native
 
         // Note that you may need to specify the SYNCHRONIZE bit as well
         // to be able to open a synchronization primitive.
-        internal const int SEMAPHORE_MODIFY_STATE = 0x00000002;
         internal const int EVENT_MODIFY_STATE = 0x00000002;
         internal const int MUTEX_MODIFY_STATE = 0x00000001;
```

Comment:
`MUTEX_MODIFY_STATE` looks unused too

Reply:
I will have another PR to move waithandle.cs to shared. I will remove this in that